modules/brew/brew.sh

```diff
@@ -277,4 +282,64 @@ if [[ "${BREW_ANALYTICS}" == false ]]; then
   fi
 fi
 
+# Create directory for brew configuration
+mkdir -p /usr/share/bluebuild/brew
+
+# Create repo-info.yml file with install packages if specified
+if [[ -n "${INSTALL_PACKAGES}" ]]; then
+    echo "install:" > /usr/share/bluebuild/brew/repo-info.yml
+    echo "${INSTALL_PACKAGES}" | sed 's/^/  - /' >> /usr/share/bluebuild/brew/repo-info.yml
+    echo "The following Brew packages will be installed when the system is live:"
+    echo "${INSTALL_PACKAGES}" | sed 's/^/  - /'
+
+    # Write brew-packages-setup script
+    cat > /usr/bin/brew-packages-setup <<EOF
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Source the Brew environment
+eval "\$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+
+# Install the Brew packages
+if [[ -n "\${INSTALL_PACKAGES[@]}" ]]; then
+  echo "Installing Brew packages..."
+  for package in "\${INSTALL_PACKAGES[@]}"; do
+    if ! brew list --formula "\$package" &> /dev/null; then
+      brew install "\$package"
+    else
+      echo "Package \$package is already installed."
+    fi
+  done
+else
+  echo "No Brew packages specified for installation."
+fi
+EOF
+
+    chmod +x /usr/bin/brew-packages-setup
+
+    # Write brew-packages-setup service
+    cat > /usr/lib/systemd/system/brew-packages-setup.service <<EOF
+[Unit]
+Description=Setup Brew Packages
+After=brew-setup.service
+Requires=brew-setup.service
+ConditionPathExists=!/var/lib/brew-packages-setup.stamp
```

Comment:
This condition makes this execute only once.

It should execute every boot, to install packages from updated install list. If there are no packages, it will just say `Package \$package is already installed.`

To make logs more sane in this scenario, instead of printing this for every package, it should print `Installed packages are up-to-date with install list, no changes need to be done`